src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -131,6 +131,15 @@ class ReportActionItemMessageEdit extends React.Component {
         Report.saveReportActionDraft(this.props.reportID, this.props.action.reportActionID, '');
         toggleReportActionComposeView(true, this.props.isSmallScreenWidth);
         ReportActionComposeFocusManager.focus();
+
+        // Scroll to the last comment after editing to make sure the whole comment is clearly visible in the report.
+        // This only applies to the last/latest comment in the report.
+        if (this.props.index === 0) {
```

Comment:
Looks good to me, @dangrous are you okay with it?

Reply:
I think that works! Not sure if we need line 136 (I think just the first line is fine) but that's very much NAB - more info probably can't hurt

Reply:
I think keeping it is fine

Reply:
Let's remove it, @priyeshshah11! I too kind of feels it's unnecessary!

Reply:
@dangrous Once the change is done we are good to merge this one!

Reply:
@Santhosh-Sellavel @dangrous Ok, removed as you both feel so.